javascript/chubbyts-uwebsockets/tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2019",
-    "module": "commonjs",
+    "erasableSyntaxOnly": true,
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
     "outDir": "./",
+    "skipLibCheck": true,
     "strict": true,
-    "declaration": true
+    "target": "esnext",
```

Comment:
production ready ?

Reply:
I think your answered in the next question. I was wondering if it is ok to go with this option in prodcution